From 696754b9332922a00e977903d6df3def6ddb3fb6 Mon Sep 17 00:00:00 2001
From: Sergey Poznyakoff <gray@gnu.org>
Date: Fri, 10 Nov 2017 21:28:17 +0200
Subject: Minor fix in MH testsuite

Running fmtcheck without .mh_profile triggers mh_install, which
surprises the testsuite with unexpected output "I'm going to
create the standard MH path for you." Defining the MH envar to
an existing file name suppresses it.

* mh/tests/fmtcnd00.at: Always define MH prior to starting fmtcheck.
* mh/tests/fmtcomp.at: Likewise.
* mh/tests/fmtfunc.at: Likewise.
---
 mh/tests/fmtcnd00.at | 2 +-
 mh/tests/fmtcomp.at  | 4 ++--
 mh/tests/fmtfunc.at  | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/mh/tests/fmtcnd00.at b/mh/tests/fmtcnd00.at
index 507e8e5..f674d41 100644
--- a/mh/tests/fmtcnd00.at
+++ b/mh/tests/fmtcnd00.at
@@ -28,7 +28,7 @@ m4_pushdef([FMTCOND],[
 AT_SETUP([$1])
 AT_KEYWORDS([format fmtcond fmtcnd00])
 AT_DATA([msg],[$2])
-AT_CHECK([fmtcheck -format 'FORMAT' msg
+AT_CHECK([MH=/dev/null fmtcheck -format 'FORMAT' msg
 ],
 [0],
 [$3])
diff --git a/mh/tests/fmtcomp.at b/mh/tests/fmtcomp.at
index 72cd823..615470a 100644
--- a/mh/tests/fmtcomp.at
+++ b/mh/tests/fmtcomp.at
@@ -22,7 +22,7 @@ dnl
 m4_pushdef([FMTCOMP],[
 AT_SETUP([$1])
 AT_KEYWORDS([format fmtcomp])
-AT_CHECK([fmtcheck -disass -dump -format '$2'
+AT_CHECK([MH=/dev/null fmtcheck -disass -dump -format '$2'
 ],
 [m4_if([$4],[],[0],[1])],
 [$3],[$4])
@@ -296,7 +296,7 @@ m4_pushdef([FMTCOMP],[
 AT_SETUP([$1])
 AT_KEYWORDS([format fmtdiag])
 AT_DATA([input.fmt],[$2])
-AT_CHECK([fmtcheck -form input.fmt
+AT_CHECK([MH=/dev/null fmtcheck -form input.fmt
 ],
 [1],
 [],
diff --git a/mh/tests/fmtfunc.at b/mh/tests/fmtfunc.at
index 844a777..0829d45 100644
--- a/mh/tests/fmtfunc.at
+++ b/mh/tests/fmtfunc.at
@@ -212,7 +212,7 @@ AT_DATA([input.fmt],[%(getenv MH_TEST)
 AT_DATA([message],[
 
 ])
-AT_CHECK([MH_TEST=defined fmtcheck -form input.fmt -width 80 message
+AT_CHECK([MH=/dev/null MH_TEST=defined fmtcheck -form input.fmt -width 80 message
 ],
 [0],
 [defined
-- 
cgit v1.0-41-gc330

